iac-extensions/docker/src/test/resources/checks/SecretsGenerationCheck/sshpass.dockerfile

```diff
@@ -8,6 +8,10 @@ FROM ubuntu:22.04
 
 ARG PASSWORD
 
+#tmp
+# Noncompliant@+1
+RUN sshpass "-p" "${PASSWORD}" ssh user@hostname
+
```

Comment:
Did you forget to remove this one? I see a #tmp comment and it seems you added the same test at line 35.

Reply:
You right, it was for debugging